oauthenticator/globus.py

```diff
@@ -193,10 +204,27 @@ def authenticate(self, handler, data=None):
         return {
             'name': username,
             'auth_state': {
-                'globus_data': globus_data,
+                'client_id': self.client_id,
+                'tokens': {
+                    tok: v for tok, v in tokens.by_resource_server.items()
+                    if tok not in self.exclude_tokens
+                },
```

Comment:
Moving the `auth_state` data down into the return dict was mostly a stylistic choice. Its only purpose is being returned in `auth_state` so removing it from the identity provider lines makes the code slightly more readable.